R/to_alias.R

```diff
@@ -276,7 +276,7 @@ from_name_to_alias <- function() {
 #' x <- c(a = 1)
 #' check_crucial_names(x, "a")
 #' try(check_crucial_names(x, "bad"))
-#' @noRd
+#' @export
 check_crucial_names <- function(x, expected_names) {
```

Comment:
If you export it, then place it in R/check_crucial_names.R, call `use_test()` and add [the tests](https://github.com/RMI-PACTA/r2dii.match/blob/main/tests/testthat/test-check_crucial_names.R).

> A reasonable starting position is to make a new .R file for each user-facing function in your package and name the file after the function -- Jenny Bryan at https://r-pkgs.org/whole-game.html#use_r

Reply:
I added the test -- so I'm ready to approve.